app-text/sigil/sigil-1.0.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+PYTHON_REQ_USE="tk"
+
+inherit cmake-utils python-single-r1 xdg-utils
+
+DESCRIPTION="Multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/ https://github.com/Sigil-Ebook/Sigil"
+SRC_URI="https://github.com/Sigil-Ebook/Sigil/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+ Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+plugins system-mathjax"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	app-text/hunspell:=
+	dev-libs/libpcre:3=[pcre16]
+	$(python_gen_cond_dep \
+		'dev-python/css-parser[${PYTHON_MULTI_USEDEP}]
+		dev-python/lxml[${PYTHON_MULTI_USEDEP}]
+		dev-python/six[${PYTHON_MULTI_USEDEP}]')
+	>=dev-qt/qtconcurrent-5.12:5
+	>=dev-qt/qtcore-5.12:5
+	>=dev-qt/qtgui-5.12:5
+	>=dev-qt/qtprintsupport-5.12:5
+	>=dev-qt/qtwebengine-5.12:5[widgets]
+	>=dev-qt/qtwidgets-5.12:5
+	sys-libs/zlib[minizip]
+	plugins? ( $(python_gen_cond_dep \
```

Comment:
That's how I've also understood how to use python-single-r1 now, but I don't see function for this USE flag in this ebuild?
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs

Reply:
Thanks for the reply.
If I understand correctly the example given there, (I will quote):
`... except under very specific circumstances, such as a package being nonfunctional unless at least one of a set of RDEPs is installed ...`
All those packages are needed for plugins support and I think a USE flag is a cleaner version.

Reply:
I always decoded it as IUSE="engine1 engine2", and `optfeature` can print multiple packages per feature, but yes I agree this is much cleaner.